umpleonline/scripts/umple_action.js

```diff
@@ -2039,109 +2041,70 @@ Mousetrap.bind(['shift+right'], function(e){
 
 //Functions for changing diagram view - change consistently in umple_page.js
 Mousetrap.bind(['ctrl+e'], function(e){
-  if(jQuery('.focus').length != 0)
```

Comment:
Good question. This needs a thorough manual test. I suspect there might be something working differently. I will do the manual test, but it may be a day or two.